### PR TITLE
Closes #233: Upgrade vagrant stack

### LINKED ISSUE #233: Upgrade Vagrant to use PHP 7.0 and Ubuntu 16.04 Xenial
Dear Yii developers,

Can you upgrade the Vagrant files to use Ubuntu 16.04 instead of Ubuntu 14.04. Furthermore use PHP 7 version instead of 5 version.

Unfortunately I am not Vagrant expert, I tried to create/modify the existing Vagrant files but I do not have success to use Ubuntu 16.04 and PHP 7 version in my projects.

### PR DESCRIPTION
**Pull request by rugabarbo**
* Ubuntu 16.04 Xenial
* PHP 7.0
* MySQL 5.7

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -25,7 +25,7 @@ end
 # vagrant configurate
 Vagrant.configure(2) do |config|
   # select the box
-  config.vm.box = 'ubuntu/trusty64'
+  config.vm.box = 'bento/ubuntu-16.04'
```
Comment (samdark) [inline Vagrantfile]: Why not `ubuntu/xenial64`?

Comment (rugabarbo) [inline Vagrantfile]: See https://github.com/mitchellh/vagrant/issues/7155

> the boxes published by canonical under the ubuntu namespace for 16.04 are very broken. They do not follow the recommended guides for how to build a Vagrant box, and they are missing key components such as guest additions, required packages, or they do things like hardcode the hostname to make running concurrent VMs impossible. ... In general, users have had more success with the boxes under the bento namespaces. A common misconception is that a namespace like "ubuntu" represents the canonical space for Ubuntu boxes. This is untrue.

Comment (rugabarbo) [inline Vagrantfile]: For example, in that moment ubuntu/xenial64 is broken: https://bugs.launchpad.net/cloud-images/+bug/1569237

Comment (samdark) [inline Vagrantfile]: OK. Thanks for info.

**Comment by samdark** [comment]
Looks good to me.

**Comment by samdark** [comment]
Thank you! Merged.

**Pull request merged**